Userland/Libraries/LibWeb/CSS/StyleValues/RotationStyleValue.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2024, Steffen T. Larssen <dudedbz@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <LibGfx/Vector3.h>
+#include <LibWeb/CSS/Angle.h>
+#include <LibWeb/CSS/StyleValues/CSSUnitValue.h>
+
+namespace Web::CSS {
+
+class RotationStyleValue : public CSSUnitValue {
+public:
+    static ValueComparingNonnullRefPtr<RotationStyleValue> create(Angle angle, float x, float y, float z)
+    {
+        return adopt_ref(*new (nothrow) RotationStyleValue(move(angle), x, y, z));
+    }
+    virtual ~RotationStyleValue() override;
+
+    Angle const& angle() const { return m_angle; }
+    FloatVector3 const& rotation_vector() const { return m_rotation_vector; }
+
+    // FIXME: Return the correct things here...
+    virtual double value() const override { return m_angle.raw_value(); }
+    virtual StringView unit() const override { return m_angle.unit_name(); }
+
+    virtual String to_string() const override;
+
+    bool equals(CSSStyleValue const& other) const override;
+
+private:
+    explicit RotationStyleValue(Angle angle, float x, float y, float z);
+
+    Angle m_angle;
+    FloatVector3 m_rotation_vector;
```

Comment:
Let's not add new places that don't support `calc()`. Keep the angle and axis in CSSStyleValue form - maybe the axis can always be 3 values if that's the most convenient.